### ISSUE TITLE
Pair operator `=>` is not dottable

### ISSUE THREAD

**Issue by mbauman** [JuliaLang/julia#27446]
Seems like we should be able to broadcast `=>` with `.=>` (from Kevin Bonham on Slack):

```julia
a = ["foo", "bar", "baz"]
b = [1,2,3]

d = Dict(a .=> b) #?
```

Instead, it parses the `.=` first — as `a .= (>b)` — so you get a:

```
ERROR: syntax: ">" is not a unary operator
```

**Comment by JeffBezanson** [comment]
Should be doable just by adding `.=>` to `prec-pair` at the top of julia-parser.scm.

**Comment by kescobo** [comment]
Neat, thanks for highlighting this @mbauman

**Comment by kmsquire** [comment]
Closed by #27447
